views/js/controller/runner/runner.js

```diff
@@ -89,6 +91,9 @@ define([
                 plugins : config && config.providers && Object.keys(config.providers.plugins)
             });
 
+            const rtl = [ ... locale.getConfig().rtl || [] ].includes(context.locale);
+            $('.delivery-scope').toggleClass('rtl', rtl);
```

Comment:
**nitpick:** The result is there, and we have the RTL flag.
However:
- The RTL detection should be done by a helper inside `util/locale`. In the current state, the code is relying on internal structure that may change for any reason. It is better to rely on a public contract. For instance, we could have a `isLanguageRTL(lang)` helper or a more generic `getLanguageDirection(lang)` in `util/locale`.
- It is enough to set a CSS class to switch the direction? Maybe this also needs the `dir` attribute to be set as well